src/CommunityToolkit.Maui.Core/Essentials/Badge/Android/DefaultBadgeProvider.android.cs

```diff
@@ -13,20 +13,20 @@ public void SetCount(uint count)
 		var packageName = Application.Context.PackageName;
 		if (packageName is null)
 		{
-			Trace.WriteLine("Unable to get package name");
+			Trace.TraceError("Unable to get package name");
```

Comment:
Please do not use `Trace.Error` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception.